### PR TITLE
7 add MRP_load computing

### PR DESCRIPTION
**Pull request by bealdav**
WIP
depends #18 and #17

### CODE REVIEW

File: mrp_load/workcenter_view.xml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                    attrs="{'invisible': [('online', '=', False)]}"
+                    type="object"/>
+            <field name="online" invisible="1"/>
+        </field>
+        <xpath expr="//tree[@ string='Work Center']" position="attributes">
+            <attribute name="colors">red:load>0 and online==False;grey:online==False</attribute>
+        </xpath>
+    </field>
+</record>
+
```
Comment (Viggor) [inline mrp_load/workcenter_view.xml:33]: Maybe you should add a button next to workcenter in tree view for notice people they can change it easily.

**Comment by Viggor** [comment]
Maybe you should add demo data for workcenter groups, because the first time you open workcenters after install this module you have a 'undefined' group, it's a bit confusing.

**Comment by Viggor** [comment]
:+1: it's good

**Comment by pedrobaeza** [comment]
This is still red and very old, so I'm closing. If you are still interested in this, feel free to reopen it fixing Travis status

**Pull request closed without merging**